dissect/target/plugins/os/unix/config.py

```diff
@@ -188,28 +376,16 @@ def __init__(
         path: str,
         entry: FilesystemEntry,
         parser_items: Optional[Union[dict, Any]] = None,
-        collapse: Optional[Union[bool, set]] = None,
     ) -> None:
         super().__init__(fs, path, entry)
-        if parser_items is None:
-            self.parser_items = self.parse_config(entry, collapse)
-        else:
-            self.parser_items = parser_items
-
-    def parse_config(self, entry: FilesystemEntry, collapse: Optional[Union[bool, set]] = None) -> ConfigParser:
-        extension = entry.path.rsplit(".", 1)[-1]
-
-        known_file = KNOWN_FILES.get(entry.name, Default)
-        parser = CONFIG_MAP.get(extension, known_file)(collapse)
-
-        with entry.open() as fh:
-            open_file = io.TextIOWrapper(fh, encoding="utf-8")
-            parser.read_file(open_file)
+        self.parser_items = parser_items
 
-        return parser
-
-    def get(self, path: str) -> ConfigurationEntry:
+    def get(self, path: Optional[str] = None) -> ConfigurationEntry:
         # Check for path in config entry
+        if not path:
+            # Return self if configuration was found.
```

Comment:
```suggestion
            # Return self if configuration was found
```
Small nitpick, sorry